### PR TITLE
0.29.2

### PR DESCRIPTION
**Pull request by lasote**
Changelog: Feature: Compatibility with Conan 1.29

### CODE REVIEW

File: cpt/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 six>=1.10.0
-conan>=1.7.0, <1.19.0
+conan>=1.7.0, <=1.19.0
```
Comment (Croydon) [inline cpt/requirements.txt]: @lasote Shouldn't this have been `< 1.20.0` to allow patch versions from 1.19.x?

Comment (uilianries) [reply cpt/requirements.txt]: https://github.com/conan-io/conan-package-tools/pull/411

Comment (uilianries) [reply cpt/requirements.txt]: anyway, good catch @Croydon ! I just saw that because a report!

Comment (Croydon) [reply cpt/requirements.txt]: Well, my Conan was suddenly downgraded when I locally installed bincrafters-package-tools (which in turns installs conan-package-tools), so I knew something was off... 👁

**Pull request closed without merging**